### ISSUE TITLE
Better approach for suggesting fee / lowest fee

### ISSUE THREAD

**Issue by luclu** [ethereum/meteor-dapp-wallet#214]
As stated in https://github.com/ethereum/mist/issues/770 users experience transactions to vanish after being broadcasted.

There are two known reasons for this to happen:

1. The **node is not synced** to the network's height - so the transaction might bounce from the relayers.
2. The **fee is to low** - so no miner will ever pick the transaction up. This is of course not intuitive as the possibility of such a low fee is propagated on the UI.

---

Dynamically fetched limits for the fee would be a great addition.

**Comment by nic-f** [comment]
No. The node was definitely synced. I waited for it to sync. A pushed the fee up (to the right away from center) a little bit so I don't think it was low fees. The transaction appeared on the list for a short while in the latest transaction list but now it does not appear.

**Comment by nic-f** [comment]
Help me recover my 41 ether and I'll give you 20 ether. I have no idea so you'll need to ask me questions.
